platform/ui/src/components/SidePanel/SidePanel.tsx

```diff
@@ -6,35 +6,21 @@ import { useTranslation } from 'react-i18next';
 import Icon from '../Icon';
 import Tooltip from '../Tooltip';
 
+type StyleMap = {
+  open: {
+    left: { marginLeft: string };
+    right: { marginRight: string };
+  };
+  closed: {
+    left: { marginLeft: string };
+    right: { marginRight: string };
+  };
+};
 const borderSize = 4;
-const expandedWidth = 248;
 const collapsedWidth = 25;
 const closeIconWidth = 30;
 const gridHorizontalPadding = 10;
 const tabSpacerWidth = 2;
-const gridAvailableWidth = expandedWidth - closeIconWidth - gridHorizontalPadding;
-
-const baseStyle = {
-  maxWidth: `${expandedWidth}px`,
-  width: `${expandedWidth}px`,
-  // To align the top of the side panel with the top of the viewport grid, use position relative and offset the
-  // top by the same top offset as the viewport grid. Also adjust the height so that there is no overflow.
-  position: 'relative',
-  top: '0.2%',
-  height: '99.8%',
-};
-
```

Comment:
I noticed that these have been deleted. Is there any particular reason for that? I believe the top and height aspects are quite significant.

Reply:
Hello, there's no reason to, it was a mistake.